all.sh

```diff
@@ -1,10 +1,42 @@
 #!/usr/bin/env bash
 
+usage() {
+  echo "Usage: $0 [--help] [-k FILTER]"
+  echo "  --help          Display this help message"
```

Comment:
there's no such option (i.e. `--help`), there's only `-h`:

```
$ ./all.sh --help
Invalid option: -
(...)
```

You can fix usage or figure out how to support short+long options.